backend/src/main/java/com/htecgroup/skynest/model/dto/CompanyDto.java

```diff
@@ -12,7 +13,10 @@ public class CompanyDto implements Serializable {
   private LocalDateTime createdOn;
   private LocalDateTime modifiedOn;
   private LocalDateTime deletedOn;
+  private String pib;
   private String name;
   private String address;
-  private String domain;
+  private String phoneNumber;
+  private String email;
+  private TierEntity tier;
```

Comment:
Entity should never be exposed via network.. Maybe create new model such as TierDto or something like that and extract just the attributes that are relevant